nginx/etc/nginx/conf.d/default.conf

```diff
@@ -1,5 +1,6 @@
 # default Docker DNS server
-resolver 127.0.0.11;
+resolver 127.0.0.11 10.89.0.1;
```

Comment:
Should we add a comment that `10.89.0.1` is for podman? At least to me as a non-podman-user, this is not obvious :)

Reply:
Yeah you're right! Will add it to the comment above